mkdocs.yml

```diff
@@ -0,0 +1,48 @@
+site_name: Backfila
+repo_name: Backfila
+repo_url: https://github.com/cashapp/backfila
+site_description: "Backfila: Service that manages backfill state, calling into other services to do batched work"
+site_author: Square, Inc.
+remote_branch: gh-pages
+
+copyright: 'Copyright &copy; 2019 Square, Inc.'
+
+theme:
+  name: 'material'
+  favicon: img/favicon.ico
+  logo: img/cashapp.png
```

Comment:
backfila.png?

Reply:
Nice, will add 👍

Reply:
I'll do this in a follow up, better to land this and unbreak CircleCI

Reply:
🤔